### DISCUSSION TITLE
Failed to get payment installment in Checkout Bricks

### DISCUSSION THREAD

**Discussion by ferpintado** [mercadopago/sdk-js#120] [Q&A] [answered]
I am trying to integrate Checkout Bricks into my application. 
While using TEST credentials the preference is created correctly however, when I render the brick and try to enter the credit card details throws an error:

```
{
    "type": "non_critical",
    "cause": "get_payment_installments_failed",
    "message": "Failed to get payment installments"
}
```

The network request hits `https://api.mercadopago.com/v1/payment_methods/installments?public_key=TEST-XXXXX&locale=es-MX&js_version=2.0.0&referer=http%3A%2F%2Flocalhost%3A3000&bin=40755957&processing_mode=aggregator&payment_type_id=&amount=699`

And its response is:

```
{
    "message": "invalid request",
    "error": "bad_request",
    "status": 400,
    "cause": []
}
```

Please advise

**Comment by icaldana** [comment, marked as answer]
Hello @ferpintado !
We apologize for the inconvenience it is causing on your workflow. This is a known-issue in which **test** keys aren't functioning, we are already working on it, and hopefully it will be fixed really soon!

Thank you very much for your contact!

**Reply by ferpintado** [reply]
Thanks, @icaldana. Is there an ETA on when this would be fixed?

**Reply by icaldana** [reply]
Hopefully by the end of Wednesday 11th.

**Reply by AdaltonLeite** [reply]
Hello @ferpintado! 

Our team has just released a hotfix to fix this bug. You can now go back to using the test environment normally.

**Reply by ferpintado** [reply]
It is working now! :) 

Thank you guys for your support.
